src/sass/sections/_conversation.scss

```diff
@@ -63,6 +63,9 @@
         .message-quote {
             @include word-wrap;
             max-width: 85%;
+            max-height: 10.4em;
```

Comment:
`10.4em` looks awkward in Firefox (half of the line is cut off) but `9.4em` works well enough for me in both Chrome and Firefox. Can you confirm that?

Reply:
I thought the cut off text would indicate that the whole text is being quoted and not just the text that is being shown. But without being able to scroll through the text it does indeed look a bit awkward. `9.4em` works for me in Chrome, Firefox and Safari.